dali/operators/audio/nonsilence_op.cc

```diff
@@ -19,38 +19,45 @@
 namespace dali {
 
 DALI_SCHEMA(NonsilentRegion)
-                .DocStr(R"code(The operator performs leading and trailing silence detection in an audio buffer.
-The operator returns the beginning and length of the non-silent region by comparing short term power of the signal
-with a silence cut-off threshold. The signal is consider silence when ``short_term_power_db < cutoff_db`` with::
+  .DocStr(R"code(Performs the leading and trailing silence detection in an audio buffer.
+
+The operator returns the beginning and length of the non-silent region by comparing the
+short term power of the signal with a silence cut-off threshold. The signal is considered to
+be silent when the ``short_term_power_db`` is less than the ``cutoff_db`` width::
 
   short_term_power_db = 10 * log10( short_term_power / reference_power )
 
-and ``reference_power`` being typically the maximum of the signal, unless specified otherwise.
-
-Inputs/Outputs
-  **Input 0** - 1D audio buffer
-  **Output 0** - Begin index of nonsilent region
-  **Output 1** - Length of nonsilent region
-
-Remarks
-  - If ``Outputs[1] == 0``, ``Outputs[0]`` value is undefined
-)code")
-                .NumInput(1)
-                .NumOutput(detail::kNumOutputs)
-                .AddOptionalArg("cutoff_db",
-                                R"code(The threshold [dB], below which everything is considered as silence)code",
-                                -60.f)
-                .AddOptionalArg("window_length", R"code(Size of a sliding window.
-The sliding window is used to calculate short-term power of the signal.)code", 2048)
-                .AddOptionalArg("reference_power",
-                                R"code(The reference power used for converting signal to db.
-If ``reference_power`` is not provided, the maximum of the signal will be used as the reference power)code",
-                                0.f)
-                .AddOptionalArg("reset_interval",
-                                R"code(The number of samples after which the moving mean average is
-recalculated to avoid loss of precision. If ``reset_interval == -1`` or the input type allows exact calculation,
-the average won't be reset. The default value should fit most of the use cases.)code",
-                                8192);
+Unless specified otherwise, ``reference_power`` is typically the maximum of the signal.
+
+Here is a list of the inputs and outputs:
```

Comment:
```suggestion

Inputs and outputs:
```
Hopefully we can insert an empty line. I think the original header was better.

Reply:
Done